samples/python/sqlalchemy-sample/test_update_singer.py

```diff
@@ -0,0 +1,18 @@
+""" Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from sample import update_singer
+
+update_singer("123-456-789", "Newfirstname", "Newlastname")
```

Comment:
If you used pytest (or Python unittest, etc), I'd recommend consolidating some of these tests.  Have several functions per file rather than just one test case per file.  You already have separate functions so I think the educational/readability value of having examples separated out by file is somewhat limited in this case.

Reply:
See also above. Splitting the tests out across multiple files make them easier to run separately. Currently, each of these files correspond to one test case in Java, which can then be executed as a single unit. An alternative could be to combine them into one file and include the function name in the command line arguments that we send to Python to look up the function that should be called.